src/components/app/MenuButtons/MetaInfo.css

```diff
@@ -15,3 +15,51 @@
   text-overflow: ellipsis;
   white-space: nowrap;
 }
+
+.metaInfoPanel {
+  width: 350px;
+  font-size: 12px;
+  line-height: 20px;
+}
+
+.metaInfoPanel,
+.metaInfoPanel .arrowPanelArrow::before {
+  background-color: white;
+}
+
+.metaInfoSection {
+  margin-top: 7px;
+  margin-bottom: 12px;
+}
+
+.metaInfoLabel,
+.metaInfoWideLabel {
+  display: inline-block;
+  width: 120px;
+  color: var(--grey-50);
+}
+
+.metaInfoWideLabel {
+  width: 140px;
+}
+
+.visualMetricsLabel {
+  display: inline-block;
+  width: 145px;
+  color: var(--grey-50);
+}
+
+.metaInfoList {
+  padding: 0;
+  margin: -19px 0 0 122px;
+  list-style: circle inside;
+}
+
+.metaInfoSubTitle {
+  padding-top: 4px;
+  padding-bottom: 4px;
+  border-bottom: 1px solid rgba(0, 0, 0, 0.05);
+  margin: 0;
+  font-size: 1.2em;
+  font-weight: 700;
+}
```

Comment:
Probably we should use one of the photon or protocol styles for this, that will be for another time.

Reply:
I believe we should only have photon in the profiler UI, and only protocol in the webpage part.

Reply:
yeah that's how I see this too.